File: 0070-climbing-stairs/solution.java
class Solution {
    public int climbStairs(int n) {
        int n1 = 0;
        int n2 = 1;
        if(n==0){
            return 0;
        }    
        for(int i=0;i<n;i++){
            int c = n1+n2;
            n1=n2;
            n2=c;
        }
        return n2;
    }
}

File: 0070-climbing-stairs/solution.c
int climbStairs(int n) {
    int arr[n+1];
    arr[0]=1;
    arr[1]=1;
    for(int i=2;i<=n;i++){
        arr[i]=arr[i-2]+arr[i-1];
    }
    return arr[n];
}
